docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.7'
+services:
+  nuxt:
+    image: kozr/lhd2020_nuxt
+    command: yarn run dev
+    working_dir: /usr/src/nuxt-app
+    volumes:
+      - .\:/usr/src/nuxt-app
```

Comment:
`.\` doesn't work on unix machines, like macs, use just `.` instead

Reply:
Wish I could join u guys for dinner instead of doing hwk and stuff ;(